GNUmakefile

```diff
@@ -12,13 +12,19 @@ debug :
 	cd src; cmake -DCMAKE_BUILD_TYPE=Debug .; make -j; make reg
 
 
+release-ulinux :
```

Comment:
I'm not 100% sure why we have two targets here, release-ulinux vs. release. Why would we choose to use release anymore after this change, since we NEED the ulinux stuff to work properly?

Is there a use case for release? If not, can't the release-ulinux changes be put under release?